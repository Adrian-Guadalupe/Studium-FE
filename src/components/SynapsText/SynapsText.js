import React from 'react';
import styled from 'styled-components';
import Text from '../../images/synapsText.png';

export const SynapsText = props => {
  return ( <TextDiv>
    <img src={ Text } alt={"synaps"}/>
  </TextDiv> );
};

SynapsText.propTypes = {};

const TextDiv = styled.div`
  font-family: Source Sans Pro;
  font-style: normal;
  font-weight: bold;
  font-size: 85px;
  line-height: 104.9%;
  /* or 128px */
  color: #231f20;
`;

