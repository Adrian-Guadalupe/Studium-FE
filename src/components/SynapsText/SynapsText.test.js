import React from 'react';
import renderer from 'react-test-renderer';

import SynapsText from './SynapsText.js';



test('StyledSynapsText render correctly', () => {
    const tree = renderer.create(<SynapsText />).toJSON();
    expect(tree).toMatchSnapshot();
});