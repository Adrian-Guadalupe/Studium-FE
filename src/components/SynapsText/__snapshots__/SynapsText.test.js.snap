// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledSynapsText render correctly 1`] = `
.c0 {
  font-family: Source Sans Pro;
  font-style: normal;
  font-weight: bold;
  font-size: 85px;
  line-height: 104.9%;
  color: #231f20;
}

<div
  className="c0"
>
  <img
    alt="synaps"
    src="synapsText.png"
  />
</div>
`;
